### ISSUE TITLE
ipsec backend not available in quay.io/coreos/flannel:v0.10.0-amd64

### ISSUE THREAD

**Issue by ncabatoff** [coreos/flannel#1047]
## Expected Behavior
ipsec backend should be available.

## Current Behavior
```
I1003 22:16:36.055584       1 main.go:353] Found network config - Backend type: ipsec
E1003 22:16:36.055597       1 main.go:272] Error fetching backend: unknown backend type: ipsec
```

## Possible Solution
I haven't looked into how you build the quay.io images.  For me a locally built image (I think I did 'make image') works fine.

## Steps to Reproduce (for bugs)

1. git clone https://github.com/ncabatoff/kubernetes-playground
2. cd kubernetes-playground
3. sed -i s,master:5000/coreos/flannel,quay.io/coreos/flannel:v0.10.0-amd64, Vagrantfile
4. vagrant up

## Your Environment

* Flannel version: quay.io/coreos/flannel:v0.10.0-amd64
* Backend used (e.g. vxlan or udp): ipsec
* Etcd version: k8s.gcr.io/etcd-amd64:3.2.18
* Kubernetes version (if used): 1.11.3
* Operating System and version: Ubuntu Xenial
* Link to your project (optional): https://github.com/ncabatoff/kubernetes-playground

**Comment by girikuncoro** [comment]
I'm also facing this issue, at the moment which flannel image to use if I want to enable ipsec backend?

**Comment by benmoss** [comment]
Yeah, I think the ipsec backend just barely missed getting merged before the 0.10.0 release was cut. @tomdee any plans for a new release? The new Windows backends also haven't seen an official release yet.
